objects/cash_register.rb

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env ruby
+
+class CashRegister
+  def total
+    @value = 0.0
+  end
+
+  def showing
+    @value
+  end
+
+  def purchase
+    @value = 3.50
+  end
+
+  def showing
+    @value
+  end
+
+  def pay
```

Comment:
The `pay` method takes an argument `amount` and subtracts the amount from the total.  If the payment is > than the total, it should say "Your change is $3.25".  If the payment is < than the total, it should say "Your new total is $1.75".